fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1189,6 +1189,43 @@ ACTOR Future<Standalone<RangeResultRef>> getWorkerInterfaces (Reference<ClusterC
 	}
 }
 
+ACTOR Future<Standalone<RangeResultRef>> getConflictingKeys(
+	VectorRef<KeyValueRef> conflictingKeys,
+	ReadYourWritesTransaction* ryw,
+	KeySelector begin, 
+	KeySelector end, 
+	GetRangeLimits limits,
+	bool snapshot,
+	bool reverse
+	) {
+		// In general, if we want to use getRange to expose conflicting keys, we need to support all the parameters it provides.
+		// It is kind of difficult to take care of each corner cases of what getRange does
+		// Thus, we use a hack way here to achieve it.
+		// We create an empty RYWTransaction and write all conflicting key/values to it.
+		// Since it is RYWTr, we can call getRange on it with same parameters given to the original getRange
+		state ReadYourWritesTransaction hackTr(ryw->getDatabase());
```

Comment:
I think for this to work you need to do two more things:

1. You need to set the read version explicitly - otherwise you will fetch a read version from the cluster.
1. You need to first clear the range (something like `hackTr.clear("\xff/conflicting_keys/", "\xff/conflicting_keys0")`).  Without that, the transaction has no way of knowing whether it knows about the whole range

Reply:
Done. Thanks a lot.